units/src/androidMain/kotlin/dev/icerock/moko/units/databinding/BindingAdapters.kt

```diff
@@ -95,3 +99,40 @@ private fun getLifecycleOwnerFromContext(context: Context): LifecycleOwner {
     if (context is ContextWrapper) return getLifecycleOwnerFromContext(context.baseContext)
     throw IllegalArgumentException("context $context not implement LifecycleOwner")
 }
+
+fun RecyclerView.bindUnits(lifecycleOwner: LifecycleOwner, units: LiveData<List<UnitItem>>) {
```

Comment:
all databinding logic should be extracted to `units-databinding` artifact. and need to remove livedata dependency